doc/examle_ospd_log.conf

```diff
@@ -0,0 +1,50 @@
+### If a custom logging configuration is required, the following
```

Comment:
Maybe you could also add a link to the python documentation? not sure if some python doc fits here.